src/components/client/campaign-application/helpers/campaignApplicationFormActions.styled.tsx

```diff
@@ -28,8 +27,7 @@ export const ActionButton = styled(Button)(() => ({
   padding: theme.spacing(1, 5),
```

Comment:
It looks like a lot of repetition here. Can we do something about that? Have a button and variations for the button with only the changed styles?

Reply:
@gparlakov No, because all of the styled components wrap a different component underneath (Button, LinkButton, SubmitButton) and the styled() utility cannot take as params more than one component.

Reply:
@teodorazhelyazkova would it not work if we were to 
```ts
const common = {
  padding: theme.spacing(1,5),
  color: 'black'
 }
 
 export const ActionButton = styled(Button)(() => ({ 
    ...common,
    my-different: style-here
   })
   ```

Reply:
You are right, it works. I've changed it.